pkg/operator/sync.go

```diff
@@ -695,17 +695,23 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 				},
 			},
 			ReadinessProbe: &corev1.Probe{
+				PeriodSeconds:       10,
+				TimeoutSeconds:      5,
+				InitialDelaySeconds: 5,
 				ProbeHandler: corev1.ProbeHandler{
 					HTTPGet: &corev1.HTTPGetAction{
-						Path: "/healthz",
+						Path: "/readyz",
 						Port: intstr.Parse("healthz"),
 					},
 				},
 			},
 			LivenessProbe: &corev1.Probe{
+				PeriodSeconds:       10,
+				TimeoutSeconds:      5,
+				InitialDelaySeconds: 5,
```

Comment:
startup probe would be a good idea.  Doing so will allow the liveness to not need to worry as much about initial delay.

Reply:
Oh, that's a great idea, added one with some of the timings I was seeing in the logs. Thank!